cfgov/ask_cfpb/tests/test_search.py

```diff
@@ -101,17 +101,18 @@ def test_en_search_no_term(self, mock_ask_search):
         mock_ask_search.queryset = mock_queryset()
         response = self.client.get(reverse("ask-search-en"), {"q": ""})
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.context_data["page"], self.en_page)
-        self.assertEqual(response.context_data["page"].query, "")
-        self.assertEqual(response.context_data["page"].result_query, "")
+        response_page = response.context_data["page"]
+        self.assertEqual(response_page, self.en_page)
+        self.assertEqual(response_page.query, "")
+        self.assertEqual(response_page.result_query, "")
 
     @override_settings(FLAGS={"ASK_SEARCH_TYPOS": [("boolean", True)]})
     @mock.patch.object(SearchQuerySet, 'spelling_suggestion')
     @mock.patch.object(SearchQuerySet, 'filter')
-    def test_en_search_suggestion(self, mock_filter, mock_suggestion):
-        # AskSearch.sugggest flips search_term and suggestion when called
+    def test_en_search_suggestion(self, mock_filter, mock_search):
```

Comment:
I think you might have over-corrected here a bit — I think it's good if the mock name matches the object's method it's mocking. Here, `mock_suggestion` was fine (or `mock_spelling_suggestion` maybe too).

Reply:
Done